portfolio/src/main/webapp/style.css

```diff
@@ -8,14 +8,36 @@ body {
   width: 650px;
 }
 
+#content h1 {
+  text-align: center;
+  color: #766;
+}
+
 #selfie {
   display: block;
+  height: 500px;
```

Comment:
What's the reason for changing this order?

Reply:
The style guide says that declarations should be alphabetized.